cinema/views.py

```diff
@@ -43,6 +57,28 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        if self.action == ("list", "retrieve"):
```

Comment:
The condition `if self.action == ("list", "retrieve"):` is incorrect. The `self.action` attribute is a string, not a tuple. You should use `if self.action in ["list", "retrieve"]:` to check if the action is either 'list' or 'retrieve'.